src/renderer/src/format.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  formatUserNamesSortedByParticipation,
+  sumDurations,
+  getMainGroupOfDurations,
+  clockifyUrl,
+  type TimeEntryReportDetailedTimeEntry,
+} from './clockifyServices'
+import { durationRoundUpByHalfHour } from './helper'
+import { getTaskTimeStatus, type Task, type TimeEntry } from './clickupServices'
+
+interface Entry {
+  task: Task
+  clickupTimeEntry: TimeEntry[]
+  timeEntry: TimeEntryReportDetailedTimeEntry[]
+}
+
+export const formatReport: Record<
+  string,
+  (id: string, entry: Entry, dateRangeStart: Date, dateRangeEnd: Date) => string
+> = {
+  'Task ID': (id, entry) => (entry.task ? id : 'Flex') + '\t',
```

Comment:
vamos colocar esse \t automaticamente lá na função pra que ninguém esqueça de colocar quando for alterar aqui